tailwind.config.js

```diff
@@ -23,6 +24,18 @@ module.exports = {
         800: '#85005B',
         900: '#5C003F'
       },
+      primaryDark: {
+        25: '#2F0421',
```

Comment:
@kyleleow @siradji can we align these with LW?

Reply:
Scan is actually following figma color system, LW should be the one updating
<img width="683" alt="image" src="https://user-images.githubusercontent.com/40191153/172815775-b9f46153-5a2d-4283-8604-2109bd2f326d.png">